### PR TITLE
feat(passport-annulment): new template for Passport Annulment

### PR DESCRIPTION
**Pull request by alexdiljar**
still a WIP, going out on dev with FF 

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203956476179430

### CODE REVIEW

File: libs/clients/passports/src/lib/passportsApi.service.ts [hunk 1]
```diff
@@ -54,8 +57,8 @@ export class PassportsService {
   }
 
   private resolvePassports(passportData: IdentityDocumentResponse[]) {
-    const passportArray = passportData.map((item) => {
-      const { productionRequestID, ...passport } = item
+    const passportArray = passportData.map((passport) => {
+      // const { productionRequestID, ...passport } = item
```
Comment (thordurhhh) [inline libs/clients/passports/src/lib/passportsApi.service.ts:61]: This could be removed now, right?

**Review by thordurhhh — APPROVED** [review]
just a minor comment.
passport client lgtm.

File: libs/clients/passports/src/lib/passportsApi.service.ts (hunk 1 above)
Comment (albinagu) [reply libs/clients/passports/src/lib/passportsApi.service.ts]: yes 👍

File: libs/application/templates/passport-annulment/src/fields/DoneImage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export const DoneImage = () => {
+  return (
+    <Box display={'flex'} justifyContent={'center'} marginY={3}>
+      <School />
+    </Box>
+  )
+}
+
+export default DoneImage
```
Comment (kksteini) [inline libs/application/templates/passport-annulment/src/fields/DoneImage/index.tsx:12]: Strange to use School for passport?

File: libs/application/templates/passport-annulment/src/fields/PassportSelection/index.tsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+              disabled:
+                tag(identityDocumentData.userPassport).variant !== 'mint',
+            },
+          ],
+          onSelect: (e) => {
+            setValue('productionRequestID', e.split(',')[0])
+            setValue('passportNumber', e.split(',')[1])
+            setValue('passportName', e.split(',')[2])
+            setValue(childPassportRadio, '')
+          },
```
Comment (kksteini) [inline libs/application/templates/passport-annulment/src/fields/PassportSelection/index.tsx:124]: Random nitpick. Is something like this cleaner?
```typescript
          onSelect: (e) => {
            let [productionRequestID, passportNumber, passportName] = e.split(',')
            setValue('productionRequestID', productionRequestID)
            setValue('passportNumber', passportNumber)
            setValue('passportName', passportName)
            setValue(childPassportRadio, '')
          },
```

Comment (kksteini) [reply libs/application/templates/passport-annulment/src/fields/PassportSelection/index.tsx:124]: Especially if we add on to this functionality, it's nice to have some semantics around the positions of the split?

File: libs/application/templates/passport-annulment/src/fields/PassportSelection/index.tsx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+                tag: child.passports ? tag(child.passports?.[0]) : undefined,
+                disabled,
+              }
+            },
+          ),
+          onSelect: (e) => {
+            setValue('productionRequestID', e.split(',')[0])
+            setValue('passportNumber', e.split(',')[1])
+            setValue('passportName', e.split(',')[2])
+            setValue(userPassportRadio, '')
```
Comment (kksteini) [inline libs/application/templates/passport-annulment/src/fields/PassportSelection/index.tsx:175]: So, the same thought here?

**Review by kksteini — APPROVED** [review]

File: libs/application/templates/passport-annulment/src/fields/DoneImage/index.tsx (hunk 2 above)
Comment (albinagu) [reply libs/application/templates/passport-annulment/src/fields/DoneImage/index.tsx:12]: i need to double check which image we are going to use here, will come back with a replacement

**Review by saevarma — APPROVED** [review]
Core files LGTM

**Pull request merged**